spt.c

```diff
@@ -80,16 +80,28 @@ void
 remaining_time(int sigint)
 {
 	char buf[17];
-	if (signal(SIGUSR1, SIG_IGN) != SIG_IGN)
-		signal(SIGUSR1, remaining_time);
+	//if (signal(SIGUSR1, SIG_IGN) != SIG_IGN)
+	//	signal(SIGUSR1, remaining_time);
```

Comment:
In my test it keeps handling the signal. Maybe if it doesn't work on your side we can restore it.